src/components/iptables/scripts/pcn-iptables-clean

```diff
@@ -6,7 +6,7 @@ ret=$(echo $?)
 
 if [ $ret -ne 0 ]
     then
-        echo "pcn-iptables already closed ..."
+        echo "pcn-iptables already closed"
```

Comment:
Since this script is about deleting (cleaning) iptables.
Should we say `"pcn-iptables doesn't exist "` and `"pcn-iptables deleted"`?

Reply:
`doesn't exist` gives me the idea of something not installed, so bad message to the end user. I'd prefer `closed` instead.
`deleted` gives me the idea of throwing something away. You `close` an application, you `delete` a file. So I'd prefer to keep `closed`.